MatrixSDK/Crypto/Verification/Transactions/SAS/MXSASTransactionV2.swift

```diff
@@ -172,6 +172,8 @@ extension MXSASTransactionV2: SasListener {
                 humanReadable: cancelInfo.reason
             )
             self.state = cancelInfo.cancelledByUs == true ? MXSASTransactionStateCancelledByMe : MXSASTransactionStateCancelled
+        case .created:
```

Comment:
This state has been split out from the `.started` state, we now have different states for when we start the verification process vs when the other side starts it.

It seems that you previously used the `isIncoming` boolean to decide which state you're going into. I think that you need to move `self.state = MXSASTransactionStateOutgoingWaitForPartnerToAccept` over here and remove the branching depending on the `isIncoming` boolean.

Reply:
Yes.